src/test/librbd/test_mock_fixture.h

```diff
@@ -25,7 +25,11 @@ ACTION_P(CopyInBufferlist, str) {
 }
 
 ACTION_P2(CompleteContext, r, wq) {
+#if defined(__clang__)
+  ContextWQ *context_wq = static_cast<ContextWQ *>(wq);
```

Comment:
@dillaman 
When I did that then GNUCC started tripping...
Have to start another run in my CentOS box for that.